src/ansys/aedt/core/application/design.py

```diff
@@ -684,7 +686,7 @@ def design_name(self, new_name):
                 timeout -= timestep
                 if timeout < 0:
                     raise RuntimeError("Timeout reached while checking design renaming.")
-        self._design_name = new_name
+            self._design_name = new_name
```

Comment:
I'd leave it as it was before because if `new_name` is in `self.design_list`, you'll not update `self._design_name`. I've looked into `set_active_design` and I didn't find this update.